src/decide/Decider.java

```diff
@@ -64,7 +64,42 @@ public boolean lic9(){
     }
 
     public boolean lic10(){
-        return true;
+        /**
+         * Returns a Boolean representing if there exists at least one set of three data points separated by
+         * exactly E PTS and F PTS consecutive intervening points, respectively, that are the vertices of a
+         * triangle with area greater than AREA1. The condition is not met when NUMPOINTS < 5.
+         * 1 ≤ E PTS, 1 ≤ F PTS
+         * E PTS+F PTS ≤ NUMPOINTS−3
+         *
+         *  @return  Boolean representing if the condition is met or not.
+         */
+
+        //No code to check if conditions to check 1 ≤ E PTS, 1 ≤ F PTS, or E PTS+F PTS ≤ NUMPOINTS−3.
+        // Code assumes this is true.
+
+        if (this.numpoints < 5){ // The condition is not met when NUMPOINTS < 5
+            return false;
+        }
+
+        boolean condition_met = false;      // Boolean to keep track if condition has been met.
+        int e = this.parameters.E_PTS;      // Just a shorthand for convenience.
+        int f = this.parameters.E_PTS;      // Just a shorthand for convenience.
+        double a, b, c, s, triangle_area;   // Used to keep calculation of the area of the Triangle readable.
+
+
+        for(int i = 0; i <= this.numpoints-3-e-f && !condition_met; i++){ // Loop through all suitable points.
+
+            // Perform Horn's formula to calculate Area of Triangle
+            a = this.points[i].distanceToCoordinate(this.points[i+e]);
```

Comment:
Should be [i+e+1] to get to the point with E_PTS in between the points, right? And +2 when it's [i+e+f] so [i+e+f+2]